metrics/test_driver/common/logger/logger.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:io';
+
+/// Prints the outputs to the console.
+class Logger {
+  static bool _quiet = false;
+  static Directory _logsDirectory;
+
+  static Directory get logsDirectory => _logsDirectory;
+
+  /// Configures [Logger].
+  ///
+  /// [quiet] defines whether print any logs to console or not.
+  /// [logsDirectory] defines the directory to store logs.
+  static void setup({bool quiet, Directory logsDirectory}) {
+    _quiet = quiet;
+    _logsDirectory = logsDirectory;
+  }
+
+  /// Print the [value] as the default console output.
+  static void log(String value) {
+    if (_quiet) return;
+
+    stdout.writeln(value);
+  }
+
+  /// Prints the [value] as the error console output.
```

Comment:
```suggestion
  /// Prints the [value] to the stderr.
```
or we can replace with "standard error output stream".